draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2775,29 +2776,34 @@ https://video.example.com/k/movie1/4/013.ts
            </t>
         </list></t>
 
-         <t>Object specification<list style="empty">
-
-              <t>Property: unix-time-window<list style="empty">
-
-                 <t>Description: A UNIX epoch time window in which the trigger SHOULD be executed.</t>
+	<t>Object specification:</t>
 
-                 <t>Type: TimeWindow object using UNIX epoch timestamps (see Section 4.2.3.2 of
-                    <xref target="RFC8006"/>)
-                 </t>
-
-                 <t>Mandatory: No, but exactly one of "unixEpochWindow", "utcWindow"
-                    or "localTimeWindow" MUST be present.</t>
-              </list></t>
-              <t>Property: utc-window<list style="empty">
-
-                 <t>Description: A UTC time window in which the trigger SHOULD be executed.</t>
+	<t><list style="empty" hangIndent="3">
+        <t><list style="hanging" hangIndent="3"><t hangText="Name: unix-time-window">
+        <vspace blankLines="1"/>
+	Description: A UNIX epoch time window in which the trigger SHOULD be executed.
+        <vspace blankLines="1"/>
+        Value: TimeWindow object using UNIX epoch timestamps (see Section 4.2.3.2 of <xref target="RFC8006"/>).
+        <vspace blankLines="1"/>
+        Mandatory: No, but exactly one of "unixEpochWindow", "utcWindow"
+                    or "localTimeWindow" MUST be present.
```

Comment:
I have spotted an error from the v12 draft.  "localTImeWindow" should have been completely removed, but it appears that there are two remaining references (this one and the one below).  Would you mind drafting a change to remove these?

Reply:
Fixed